src/kohlrahbi/__init__.py

```diff
@@ -27,22 +27,26 @@
 
 
 # pylint:disable=anomalous-backslash-in-string
-def get_valid_pruefis(list_of_pruefis: list[str], all_known_pruefis: Optional[list[str]] = None) -> list[str]:
+def get_valid_pruefis(
+    pruefi_to_file_mapping: dict[str, Union[str, None]], all_known_pruefis: Optional[list[str]] = None
```

Comment:
```suggestion
    pruefi_to_file_mapping: dict[str, Optional[str]], all_known_pruefis: Optional[list[str]] = None
```
from typing import Optional.
Optional[Foo] ist das gleiche wie Union[Foo, None]